File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/controller/RpcPostmanHomeController.java
package org.javamaster.invocationlab.admin.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * 返回html文件
 *
 * @author yudong
 */
@Controller
public class RpcPostmanHomeController {

    @RequestMapping(value = "/")
    public String index() {
        return "index.html";
    }
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/controller/RpcPostmanHomeController.kt
package org.javamaster.invocationlab.admin.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

/**
 * 返回html文件
 *
 * @author yudong
 */
@Controller
class RpcPostmanHomeController {

    @RequestMapping(value = ["/"])
    fun index(): String {
        return "index.html"
    }

}
